README.md

```diff
@@ -35,8 +35,9 @@ $ sudo dnf install nfs-utils
 $ sudo mount <IP address of machine which exports /var/lib/pulp>:/var/lib/pulp /var/lib/pulp
 ```
 
-2. Configure your connection to MongoDB in /etc/pulp/settings.py. You can use the same configuration
- as you have in Pulp 2 (only seeds might need to be different, it depends on your setup).
+2. Configure your connection to MongoDB in pulp_2to3_migration/app/settings.py.
```

Comment:
I think it's only a default configuration which is provided there, we still want to tell users to configure everything in /etc/pulp/settings.py.

Reply:
i made this change because how we were reading the setting, with this issue fixed https://pulp.plan.io/issues/5425 is not longer needed, i will remove